Source/Upstream: Yes, in v238-stable branch

From 4c7f25c07692d8f5f132839e9500617d97358b7e Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Wed, 21 Mar 2018 11:46:49 +0100
Subject: [PATCH 50/51] sd-bus: drop fd_nonblock() calls that are implied by
 rearrange_stdio() (#8514)

(cherry picked from commit 68b525d1d1e8153cbc2e2354fa650aa165f7a434)
---
 src/libsystemd/sd-bus/bus-socket.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/libsystemd/sd-bus/bus-socket.c b/src/libsystemd/sd-bus/bus-socket.c
index 166fba157..082688b17 100644
--- a/src/libsystemd/sd-bus/bus-socket.c
+++ b/src/libsystemd/sd-bus/bus-socket.c
@@ -963,9 +963,6 @@ int bus_socket_exec(sd_bus *b) {
                 if (rearrange_stdio(s[1], s[1], STDERR_FILENO) < 0)
                         _exit(EXIT_FAILURE);
 
-                (void) fd_nonblock(STDIN_FILENO, false);
-                (void) fd_nonblock(STDOUT_FILENO, false);
-
                 if (b->exec_argv)
                         execvp(b->exec_path, b->exec_argv);
                 else {
-- 
2.16.3

